app/src/components/tests/example.test.js

```diff
@@ -0,0 +1,7 @@
+function sum(a, b) {
+    return a + b;
+  }
+  
+test("it should return sum", () => {
+    expect(sum(1, 2)).toBe(3);
```

Comment:
should your tests be related with ui? 
Or is it just for try?

Reply:
> should your tests be related with ui?
> Or is it just for try?
In our homework description, we have to add a script command. There are no details about UI or Inner logic.